l10n_es_facturae_face/tests/test_facturae_face.py

```diff
@@ -4,6 +4,8 @@
 import logging
 from unittest import mock
 
+import requests
```

Comment:
Why do you add the library? It does not seem to be used

Reply:
It is used in line 28